### ISSUE TITLE
Missing ncinput functions on libffi

### ISSUE THREAD

**Issue by kascote** [dankamongmen/notcurses#2591]
The new functions to check modifiers are missing on `libffi`.
only `ncinput_equal_p` and `ncinput_nomod_p` are being exported

```
❯ nm -gU libnotcurses-ffi.3.0.5.dylib | grep ncinput
00000000000041f0 T _ncinput_equal_p
0000000000004240 T _ncinput_nomod_p
```

**Comment by dankamongmen** [comment]
ugh, yeah, i knew we'd need to manually keep this up to date. wait i think @KristofferC wrote a script which will do this, one sec...

**Comment by dankamongmen** [comment]
yes! what a champion!

**Comment by dankamongmen** [comment]
thanks for the heads up @kascote ; i believe this to be fixed in master, and will be visible in 3.0.6.

**Comment by dankamongmen** [comment]
i've added code to call this script in `release.sh`, so hopefully this won't happen again.
